src/app/meshes/views/MeshOverviewView.vue

```diff
@@ -169,13 +167,33 @@ const extendedMesh = computed(() => {
   }
 })
 
-const totalPolicyCount = computed(() => store.state.sidebar.insights.mesh.policies.total)
+const totalPolicyCount = computed(() => {
+  if (meshInsights.value === null) {
+    return 0
+  }
+
+  return Object.values(meshInsights.value.policies).reduce((total, stat) => total + stat.total, 0)
+})
+const policyTypes = computed(() => {
+  if (meshInsights.value === null) {
+    return []
+  }
 
-const policyCounts = computed(() => {
-  return store.state.policyTypes.map((policyType) => ({
-    ...policyType,
-    length: meshInsights.value?.policies[policyType.name]?.total ?? 0,
-  }))
+  return Object.entries(meshInsights.value.policies)
+    .map(([policyTypeName, stat]) => {
+      const policyType = store.state.policyTypesByName[policyTypeName]
+
+      if (policyType && stat.total !== 0) {
+        return {
+          name: policyType.name,
+          path: policyType.path,
+          total: stat.total,
+        }
+      }
+
+      return null
+    })
+    .filter(notEmpty)
```

Comment:
What does `notEmpty` offer over using `Boolean`?

Reply:
Type safety. With `.filter(Boolean)`, the resulting list still has values that can be `null`. `notEmpty` returning `value is TValue` prevents this.